packages/suite/src/components/firmware/CheckSeedStep.tsx

```diff
@@ -123,6 +124,10 @@ export const CheckSeedStep = ({ deviceWillBeWiped, onClose, onSuccess }: CheckSe
 
     const { heading, description, checkbox } = getContent();
 
+    const onCheckBackup = () => {
+        dispatch(goto('recovery-index'));
```

Comment:
You'll also need to call `resetReducer` from `useFirmwareInstallation` here (before redirecting), otherwise the modal will reopen on this step instead of the first one when you open it next time.